nmap.go

```diff
@@ -94,6 +98,55 @@ func (s *Scanner) Streamer(stream io.Writer) *Scanner {
 	return s
 }
 
+// Version returns the version of the installed nmap binary
+func (s *Scanner) Version() (result any, err error) {
```

Comment:
```suggestion
func (s *Scanner) Version() (string, error) {
```

I recommend never using named return parameters when the function signature is implicitly clear enough without.